frontend/src/components/PathsViewer.svelte

```diff
@@ -91,14 +91,20 @@ async function addFolder() {
   addInstallPath(result);
 }
 
-async function addFile() {
-  const result = await App.PickTomlFile();
+function addFile() {
+  App.PickRLBotToml()
+    .then((result) => {
+      if (!result) {
+        return;
+      }
 
-  if (!result) {
-    return;
-  }
-
-  addInstallPath(result);
+      addInstallPath(result);
+    })
+    .catch((error) => {
+      toast.error(
+        "Failed to add file: Only bot.toml or script.toml files are allowed",
+      );
+    });
```

Comment:
What happens on malformed tomls?

Reply:
they get added anyways but I think it is the right thing to do

Reply:
This is what it looks like, but this isn't something that this pr introduced and is therefor imo kinda "out of scope".
![image](https://github.com/user-attachments/assets/d9211c90-f046-48af-a85e-998086508d91)

Reply:
Not a bad idea, but where would this go? Only for add file? When loading/reloading the list of bots/scripts?

Reply:
Also true, maybe we address this later